### ISSUE TITLE
Unreadable text in Add Client-Side Library dialog with dark theme

### ISSUE THREAD

**Issue by Aquastorm** [aspnet/LibraryManager#376]
### Functional impact
Minor impact: labels can't be read because they have the same color as the background (but one can guess what should be displayed).
Major impact: it is very difficult to pick a library among the list of proposed libraries because there isn't enough contrast between the libraries names (very light gray) and the drop-down list background (white).

### Minimal repro steps
1. Open _Tools -> Options..._
2. Under _Environment -> General_ select the _Dark_ color theme
3. [For this step take any ASP.NET Core project you like] Right click on your project and select _Add -> Client-Side Library..._
4. The Add Client-Side Library dialog will pop-up. Write the name of a library (for instance "boostrap") in the _Library_ text field and notice how unreadable the list of proposed libraries is (see the screenshot below)

### Expected result
Dialog labels and even more important the list of proposed libraries should be readable.

### Actual result
This is what the user sees with the Visual Studio dark theme:

![image](https://user-images.githubusercontent.com/32854043/45500042-fa200e80-b77d-11e8-9983-cb3a0bb0dfb7.png)

### Further technical details
Visual Studio Pro 2017 version 15.8.4

**Comment by justcla** [comment]
Thanks for raising the issue.
We'll look into it.

**Comment by jodavis** [comment]
It looks like this becomes an issue for our dialog when the DevOpsPackage is loaded, e.g. when using Build->Configure Continuous Delivery to Azure...

Something about the way they that package themes its dialog is causing some theme colors to show up in other unthemed dialogs.

**Comment by bdelaney** [comment]
I have project where DevOpsPackage is NOT loaded and see the same problem with Dark theme in VS 2017 15.9.4
Changing to any other theme and I can see the list properly.

**Comment by jimmylewis** [comment]
I haven't been able to repro this for a while in VS 2019 builds (after repeated attempts).  I'm guessing something was fixed in the VS shell that is preventing other packages from modifying the brushes for unthemed dialogs.  Closing for now unless we get new reports in VS 16.x.
